Cabal/Distribution/Compat/FilePath.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE CPP #-}
+
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
+
+module Distribution.Compat.FilePath
+( isExtensionOf
+, stripExtension
+) where
+
+import Data.List
```

Comment:
I'd use explicit import lists

Reply:
For `Data.List` ok, but for FilePath I'd have to use CPP in the import list which I'd rather not.

Reply:
Yes.